cpp/lib/answer/json.cpp

```diff
@@ -256,6 +255,13 @@ const JSONString& JSON::asString() const {
     if (!isString()) throw Utils::exception(DriverError::INVALID_JSON_CAST, NAME(_type), NAME(JSONType::STRING));
     return stringValue;
 }
+
+const JSONString JSON::toString() const {
+    std::stringstream ss;
+    appendToString(ss, *this);
+    return JSONString(ss.str());
+}
```

Comment:
Since we are using `sstream` anyway, an implementation of `operator<<(std::ostream&, JSON&)` would be better instead. It's going to be pretty much identical to the `appendToString()`. `toString()` then could just be:
```suggestion
const std::string JSON::toString() const {
    std::stringstream ss;
    ss << *this;
    return ss.str();
}
```